<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-2L70A79" failures="1" tests="6" name="Test Runner 2" time="0.074" errors="0" timestamp="2024-10-02T15:19:29 IST">
  <testcase classname="distributedParallel.Test4" name="demo1" time="0.012">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at distributedParallel.Test4.demo1(Test4.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- demo1 -->
  <testcase classname="distributedParallel.Test4" name="demo2" time="0.002"/>
  <testcase classname="distributedParallel.Test4" name="demo3" time="0.002"/>
  <testcase classname="distributedParallel.Test3" name="demo1" time="0.001"/>
  <testcase classname="distributedParallel.Test3" name="demo2" time="0.001"/>
  <testcase classname="distributedParallel.Test3" name="demo3" time="0.001"/>
</testsuite> <!-- Test Runner 2 -->
